### PR TITLE
Update log-files.md

### PR DESCRIPTION
**Pull request by FISHMANPET**
Added more detailed information about location of patchdownloader.log, because it won't be in the normal locations for most SCCM log files

Not sure where the log file will be for an ADR if the client isn't installed

### CODE REVIEW

**Comment by PRMerger14** [comment]
@FISHMANPET : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @FISHMANPET for the contribution! I'll review after the end of the contest.

#hold-off
(reviewers, please don't edit - I'll take care of it.)

**Comment by aczechowski** [comment]
Tagging for #MMS2019Docathon

File: sccm/core/plan-design/hierarchy/log-files.md [hunk 1]
```diff
@@ -411,7 +411,7 @@ The log file SMS_DM.log on the site system server also records communication bet
 |Log name|Description|Computer with log file|  
 |--------------|-----------------|----------------------------|  
 |objreplmgr.log|Records details about the replication of software updates notification files from a parent site to child sites.|Site server|  
-|PatchDownloader.log|Records details about the process of downloading software updates from the update source to the download destination on the site server.|Computer that hosts the Configuration Manager console from which downloads are initiated|  
+|PatchDownloader.log|Records details about the process of downloading software updates from the update source to the download destination on the site server.|When downloading updates manually, this will be located in the %temp% directory of the user running the console on the machine you're running the console. For Automatic Deployment Rules, this will be located on the site server in %windir%\CCM\Logs if the ConfigMgr client is installed on the site server|
```
Comment (aczechowski) [inline sccm/core/plan-design/hierarchy/log-files.md]: A few suggested edits for style:

```suggestion
|PatchDownloader.log|Records details about the process of downloading software updates from the update source to the download destination on the site server.|When you manually download updates, this file is in your `%temp%` directory on the computer where you use the console. For automatic deployment rules, if the Configuration Manager client is installed on the site server, this file is on the site server in `%windir%\CCM\Logs`.|  
```

FYI, if you accept my suggestion, note that our automation process may auto-close this pull request. But that's ok! I'll reopen and finalize the pull request.

**Comment by aczechowski** [comment]
Ping @FISHMANPET - can you review and revise?

**Comment by PRMerger12** [comment]
@FISHMANPET : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @FISHMANPET !
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *could* be today (Wednesday, 22 May) around 11:00 Pacific time. No further action needed from you.

**Comment by PRMerger14** [comment]
@GitHubber17 : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Pull request merged**